wgengine/userspace.go

```diff
@@ -1554,3 +1557,8 @@ func (ls fwdDNSLinkSelector) PickLink(ip netaddr.IP) (linkName string) {
 	}
 	return ""
 }
+
+var (
+	metricMagicDNSPacketIn = clientmetric.NewGauge("magicdns_packet_in") // for 100.100.100.100
+	metricForwardToOS      = clientmetric.NewGauge("packet_forward_to_os")
```

Comment:
reflect rather than forward? It's a packet we got from localhost, which we shouldn't have received, and must reflect back to the kernel for things to work correctly

Reply:
done